contracts/minter-suite/Minters/MinterDAExpRefund/MinterDAExpRefundV0.sol

```diff
@@ -0,0 +1,965 @@
+        // reduce the netPaid (in storage) to value after refund deducted
+        receipt.netPaid = amountDue;
+        // emit event indicating new receipt state
+        emit ReceiptUpdated(
+            msg.sender,
+            _projectId,
+            receipt.numPurchased,
+            receipt.netPaid
+        );
+
+        // INTERACTIONS
+        bool success_;
+        (success_, ) = _to.call{value: refund}("");
+        require(success_, "Refund failed");
+    }
+
+    /**
+     * @notice Refunds the sender's payments above current amount due for
+     * projects in `_projectIds`. The amount due is the the price paid for
+     * the most recently purchased token on each project, or the base price if
+     * the artist has withdrawn revenues after the auction reached base price.
+     * This function is callable at any point, but is expected to typically be
+     * called after auctions have sold out above base price, or after auctions
+     * have been purchased at base price. This minimizes the amount of gas
+     * required to refund the sender.
+     * Sends total of all refunds to msg.sender in a single chunk.
+     * Entire transaction reverts if any refund fails.
+     * @param _projectIds Array of project IDs to refund payments on.
+     */
+    function claimRefunds(uint256[] calldata _projectIds) external {
+        claimRefundsTo(payable(msg.sender), _projectIds);
+    }
+
+    /**
+     * @notice Refunds the sender's payments above current amount due for
+     * projects in `_projectIds`. The amount due is the the price paid for
+     * the most recently purchased token on each project, or the base price if
+     * the artist has withdrawn revenues after the auction reached base price.
+     * This function is callable at any point, but is expected to typically be
+     * called after auctions have sold out above base price, or after auctions
+     * have been purchased at base price. This minimizes the amount of gas
+     * required to refund the sender.
+     * Entire transaction reverts if any refund fails.
+     * Sends total of all refunds to address `_to` in a single chunk.
+     * @param _to Address to send refund to.
+     * @param _projectIds Array of project IDs to refund payments on.
+     */
+    function claimRefundsTo(address payable _to, uint256[] memory _projectIds)
+        public
+        nonReentrant
+    {
+        // CHECKS
+        // input validation
+        require(_to != address(0), "No refund to the zero address");
+        // EFFECTS
+        // for each project, tally up the refund amount and update the receipt
+        // in storage
+        uint256 refund;
+        uint256 projectIdsLength = _projectIds.length;
+        for (uint256 i = 0; i < projectIdsLength; ) {
+            uint256 projectId = _projectIds[i];
+            ProjectConfig storage _projectConfig = projectConfig[projectId];
+            Receipt storage receipt = receipts[msg.sender][projectId];
```

Comment:
Given how this is abstracted, I believe this should be checking `receipts[_to][_projectId]` instead, no?

Reply:
(see my response for analogous comment in `claimRefundTo` comment 🙏 )

Reply:
Ohhhhh–I misunderstood intended use here. Yep, this entirely makes sense (and is also aligned with the documentation, that's my bad). LGTM.